modules/platformioBidAdapter.js

```diff
@@ -24,7 +24,7 @@ export const spec = {
     };
   },
   interpretResponse: (response, request) => (
-    bidResponseAvailable(request, response.body)
+    bidResponseAvailable(request, response)
```

Comment:
@varashellov What you removed here was actually correct. 

First argument to interpretResponse is this
```
response = {
  body: response,
  headers: headerParser(responseObj)
};
```